Sources/Flowpack.Website/Classes/Aspects/GitHubCachingAspect.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Flowpack\Website\Aspects;
+
+/*
```

Comment:
this is the only file that contains the license header, the other files don't.
Intended?

Reply:
Yup, please add to the other files as well.